x-pack/legacy/plugins/siem/public/components/embeddables/__snapshots__/index_patterns_missing_prompt.test.tsx.snap

```diff
@@ -21,7 +21,7 @@ exports[`IndexPatternsMissingPrompt renders correctly against snapshot 1`] = `
           values={
             Object {
               "beats": <a
-                href="https://www.elastic.coguide/en/beats/libbeat/current/getting-started.html"
+                href="https://www.elastic.co/guide/en/beats/libbeat/mocked-test-branch/getting-started.html"
```

Comment:
Good catch on the snapshots. This looks like a good improvement or a bug that was caught? 

Previous version `https://www.elastic.coguide/e` looks like it was missing a slash.

Reply:
Yeah, we had some ad hoc mocks that were a little off! 👍